tests/test_build.py

```diff
@@ -844,7 +844,10 @@ def test_theme_switcher(sphinx_build_factory, file_regression) -> None:
 
 def test_shorten_link(sphinx_build_factory, file_regression) -> None:
     """Regression test for "edit on <provider>" link shortening."""
-    sphinx_build = sphinx_build_factory("base").build()
+    confoverrides = {
+        "html_theme_options": {"shorten_urls": True},
```

Comment:
I'm not 100% sure why the default value in theme.conf isn't picked up by the test build process, but being forced to set `shorten_urls` to true also makes the setting more explicity.